src/walk.js

```diff
@@ -25,7 +25,7 @@ function _walk (node, callback, o = {}, property, parent) {
 			return ret;
 		}
 
-		for (let child of children(node)) {
+		for (let child of children.get(node)) {
```

Comment:
Looking at a call like `children.get(node)` I’m not sure I intuitively understand what it does. Maybe `children.for(node)` or `children.of(node)` would be clearer?

Reply:
I like `children.of`, and I agree that it sounds better and makes sense that it would return an array

Reply:
I don't like `of` because I can't tell what it is for and it looks too much like the built-in `of`.
It only reads like something (vaguely) because of how you named `children`.

Reply:
@joyously 
Literally all of the proposed names correspond to built-in keywords. 🙃
Also, no name we go with will make sense by itself (e.g. neither does `get` or `for`), unless we prefix/suffix it with `children` (e.g. `getChildren`) which will make `children.*` calls terrible to read.

You either design APIs that look nice with `import * as foo` or APIs that are meant to be imported like `import { fn1, fn2 }`, but rarely do you get nice readability with both by default (people can always rename using `as`).

Reply:
That said, nothing preventing us from exposing e.g. *both* `of` and `childrenOf` exports, though that would be a bit tedious to keep up with.

Reply:
> Literally all of the proposed names correspond to built-in keywords. 

Yes, that's why I didn't like them either, but was commenting on the one that was chosen.
I'm not very versed in `import`s. Is the name always there or can functions be used without the import name?
Are all the functions in the module really about `children`? Or is it more like `ancestry` or `hierarchy`? I think function names should usually be verbs, so prepositions seem too vague.